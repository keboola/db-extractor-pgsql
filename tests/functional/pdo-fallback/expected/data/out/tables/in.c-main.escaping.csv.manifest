{
    "destination": "in.c-main.escaping",
    "incremental": false,
    "columns": [
        "table_schema",
        "table_name",
        "table_type"
    ]
}
