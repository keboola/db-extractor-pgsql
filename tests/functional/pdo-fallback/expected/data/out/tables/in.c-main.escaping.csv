"table_schema","table_name","table_type"
"public","escaping","BASE TABLE"
"public","escaping","BASE TABLE"
